crypto/src/main/java/org/web3j/crypto/Sign.java

```diff
@@ -108,7 +108,7 @@ private static BigInteger recoverFromSignature(int recId, ECDSASignature sig, by
         BigInteger i = BigInteger.valueOf((long) recId / 2);
         BigInteger x = sig.r.add(i.multiply(n));
         //   1.2. Convert the integer x to an octet string X of length mlen using the conversion
-        //        routine specified in Section 2.3.7, where mlen = ⌈(log2 p)/8⌉ or mlen = ⌈m/8⌉.
```

Comment:
I removed the `⌈` character because was giving me compilation errors on OSX 10.13.3 with Oracle JDK 1.8.0_162, and I end up committing this by mistake.

`:rlp:javadoc
/Users/xavi/Development/Projects/web3j/crypto/src/main/java/org/web3j/crypto/Sign.java:111: error: unmappable character for encoding ASCII
        //        routine specified in Section 2.3.7, where mlen = ???(log2 p)/8??? or mlen = ???m/8???.`

A workaround would be to add this to the  global `build.gradle`:
```
javadoc {
    options.encoding = 'UTF-8'
}
```

I'll push an additional commit fixing this.